wheat_filtration/munge.py

```diff
@@ -2,75 +2,130 @@
 import os
 import string
 
+import nltk.data
 
-def _next_punc_index(i, tokens: List[str]):
-    """Returns the index of the element in an list of tokens following the next
-    punctuation after index i. If no punctuation is found within 250 indeces after i,
-    returns the index i + 250."""
-    curr_index = i
-    while curr_index < (i + 250):
-        possible_punc = tokens[curr_index][-1]
-        if possible_punc in (".", "!", "?"):
-            return curr_index + 1
-        curr_index += 1
-    return curr_index
-
-
-def split_corpus(source: str):
-    """Splits corpus into appropriately sized documents (100-500 words).
+
+def import_corpus(source: str):
+    """Load corpus from source into one string.
     Arguments:
-        source (str): the path to the txt file or directory (containing txt files)
+        source (str): the path to the text file or directory (containing text files)
             where the corpus is located. if source is a directory, it must end in /
-            or \\ (whatever is appropriate to your system)!
+            or \\ (whatever is appropriate to your system)! This function will parse all
+            files in the given directory that are not system files.
     Returns:
-        string list: a list containing strings representing documents in the corpus.
-            Documents are between 250 and 500 words. Documents end on sentences as much
-            as possible within the wordcount limits."""
+        full_corp (str): a string containing the corpus"""
 
     full_corp = ""
-    if source[-4:] == ".txt":
-        # handles case where source is a text file
-        with open(source, "r") as inf:
-            full_corp += inf.read()
-
-    else:
-        # handles case where source is directory, or throws exception if it is not
+    try:
+        with open(source, "r") as in_file:
+            full_corp += in_file.read()
+    except IsADirectoryError:
```

Comment:
Generally, if there's a function that can help you check things without throwing an exception, that's a good idea to use instead for control flow. In this case, that function is `os.path.isdir(path)`.